adhocteam/Head-Start-TTADP
# Fix broken  paths in openapi schema

**Description of change**

There were a couple of typos in the schema that were causing Redoc to show errors. This fixes that.

**How to test**

Run the docker container for api-docs on main, then load http://localhost:5000/, see an error message. Then checkout this PR's branch and reload the URL, and docs should appear.

**Issue(s)**
* None
